confluent_kafka/src/confluent_kafka.c

```diff
@@ -836,6 +836,33 @@ static void error_cb (rd_kafka_t *rk, int err, const char *reason, void *opaque)
 	CallState_resume(cs);
 }
 
+static int stats_cb(rd_kafka_t *rk, char *json, size_t json_len, void *opaque)
+{
+	Handle *h = opaque;
+	PyObject *eo=NULL, *result=NULL;
+	CallState *cs=NULL;
+
+	cs = CallState_get(h);
+	if (json_len== 0 || !h->stats_cb) {
```

Comment:
We dont need to enable the C stats_cb if no Python stats_cb has been set, which remove the `!h->stats_cb` check